README.md

```diff
@@ -102,7 +146,14 @@ Using the object literal notation, Define an object called student and give it t
 
 Your Answer:
 ```js
-// write code here
+var student= {
+name: "Kaira Johnson",
+age:"33"
+}
+
+function sayHello() {console.log ("Hi, my name is " + [student.name])}
```

Comment:
This can technically work, but we are actually looking for a method within the object.. So something like:

```
var student= {
 name: "Kaira Johnson",
 age:"33",
sayHello: function(){
    console.log("Hi, my name is " + this.name);
  }
}
```

Let me know if you want to review methods!